app/src/shared/errors/Web3/index.jsx

```diff
@@ -2,10 +2,24 @@
 
 import { I18n } from 'react-redux-i18n'
 
+export const errorCodes = {
+    WEB3_NOT_SUPPORTED: 'WEB3_NOT_SUPPORTED',
```

Comment:
What do you think about namespacing those? I mean:
```js
{
    WEB3_NOT_SUPPORTED: 'WEB3/NOT_SUPPORTED',
    WEB3_NOT_ENABLED: 'WEB3/NOT_ENABLED',
    WALLET_LOCKED: 'WEB3/WALLET_LOCKED',
    WRONG_NETWORK_SELECTED: 'WEB3/WRONG_NETWORK_SELECTED',
    CHALLENGE_FAILED: 'WEB3/CHALLENGE_FAILED',
    CREATE_IDENTITY_FAILED: 'WEB3/CREATE_IDENTITY_FAILED',
    IDENTITY_EXISTS: 'WEB3/IDENTITY_EXISTS',
}
```

I'm wondering.. maybe it's a good idea to keep them in a single "family". It may be a good idea, but it doesn't have to, heh.

Reply:
Yep, not a bad idea!